samples/ContextualPopupSample.js

```diff
@@ -224,7 +224,7 @@ enyo.kind({
 		this.$.buttonPopup.setAutoDismiss(!inSender.getActive());
 	},
 	dismissRadioSelection: function(){
-		if(this.$.nestedRadioDismissButton.value) this.$.nestedRadioPopup.hide();
+		if(this.$.nestedRadioDismissButton.value) this.$.nestedRadioPopup.closePopup();
```

Comment:
Makes sense, if UX deems that the activator should always be hidden, we could make the change to `hide`. I do wonder if any app developers have already required use of the internal closePopup though, and if changing the default behavior for `hide` could have affects on apps currently running (possibly not?).